content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,161 @@
+   2. Hover the mouse over the :guilabel:`Phone` or :guilabel:`Mobile` field and a :guilabel:`📱
+      (phone) SMS` icon appears.
+   3. Click the :guilabel:`📱 (phone) SMS` icon and a :guilabel:`Send SMS Text Message` pop-up
+      appears.
 
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+   .. image:: in_app_purchase/send-sms.png
+      :align: center
+      :alt: The Send SMS Text Message service can be accessed from a contact's record.
 
-.. image:: in_app_purchase/image4.png
-   :align: center
+   4. Type a message in the :guilabel:`Send SMS Text Message` pop-up, then click :guilabel:`Send
+      SMS`. Odoo sends the message via SMS and log what was sent in the chatter.
+
+   The prepaid credits for the SMS service are automatically deducted from the existing pool. If
+   there are not enough credits to send the message, Odoo prompts the purchase of more.
+
+.. seealso::
+   For more information on how to use various IAP services, review the documentation below:
+
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an IAP service is used, the prepaid credits for that service are spent. Odoo prompts the
+purchase of more credits when there are not enough credits to continue using the service. Email
+alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in "packs" from the `Odoo IAP website
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - **10** credits
+   - **100** credits
+   - **500** credits
+   - **1,000** credits
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+If there are not enough credits to perform a task, the database automatically prompts the purchase
+of more credits. Users can check the current balance of credits for each service, and manually
+purchase more credits through the service's :guilabel:`Account Information` page.
+
+Manually buy credits
+~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
```suggestion
Manually buy credits
~~~~~~~~~~~~~~~~~~~~

To manually buy credits in Odoo, follow these steps:

```
a brief intro sentence like this could be beneficial as a lead-in to the following steps